### PR TITLE
202151172 Spruha Thorat

### PR DESCRIPTION
**Pull request by Spruha017**
I had updated data.json file

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -98,6 +98,15 @@
       "linkedin": "https://www.linkedin.com/in/ayushkhandelwal28/",
       "quote": "Always Learning"
     }
```
Comment (nlok5923) [inline data.json]:
```suggestion
    },
```

**Review by nlok5923 — CHANGES_REQUESTED** [review]
Please take a look at suggestion and then we gtg with this pr.

File: data.json (hunk 1 above)
Comment (Spruha017) [reply data.json]: I made the changes . Are the visible to you ?

**Comment by RahulBansal0502** [comment]
Checkout:https://iiitv.github.io/github-workshop-jan-2022/

<blockquote><div><strong><a href="https://iiitv.github.io/github-workshop-jan-2022/">GDSC IIIT-V HacktoberFest Web App
    Git and GitHub Workshop Jan 2022</a></strong></div></blockquote>

**Pull request merged**